client/src/app/shared/utils.ts

```diff
@@ -1,6 +1,9 @@
 export default class Utils {
 
   static isUrl(value: string) {
+    if (!(value.toString().startsWith('http'))) {
+      return false;
+    }
```

Comment:
I wonder if we could just use this as the main logic to check if a cell value is a URL, which means removing the try-catch below 😅

Reply:
Do we have some test cases for this code? Have there been values that caused problems? How is this functionality tested?

Reply:
@aaclan-ebi I don't think it would be enough, as any string would be a valid URL that starts with `http`.

Reply:
@amnonkhen Currently all the ontology related values are converted into a URL. Values like `NCBITaxon:10090`.
I already discuss it with Ray that the textual appearence would be just fine.
I am going to add tests for thsi. You are right that is missing.

Reply:
@amnonkhen I added unit tests to the `isUrl` utility method